src/contracts/examples/fastlane-online/SolverGateway.sol

```diff
@@ -252,14 +260,18 @@ contract SolverGateway is OuterHelpers {
         bytes32 _solverOpHash = keccak256(abi.encode(solverOp));
         uint256 _congestionBuyIn = S_congestionBuyIn[_solverOpHash];
 
+        uint256 _bidFactor = (solverOp.bidAmount ** 2) * _SLIPPAGE_BASE / (minAmountUserBuys + 1) ** 2;
+        if (_bidFactor > _GLOBAL_MAX_SLIPPAGE) _bidFactor = _GLOBAL_MAX_SLIPPAGE;
+
         score = (
             (_congestionBuyIn + (maxFeePerGas * totalGas)) // A solver typically has to pay maxFeePerGas * gas as a
                 // requirement for winning.
                 * totalGas / (totalGas + solverOp.gas) // double count gas by doing this even in unweighted score (there's
                 // value in packing more solutions)
-                * (uint256(_aData.auctionWins) + 1) / (uint256(_aData.auctionWins + _aData.auctionFails) + solverCount + 1)
-                * (solverOp.bidAmount > (minAmountUserBuys + 1) * 2 ? (minAmountUserBuys + 1) * 2 : solverOp.bidAmount)
-                / (minAmountUserBuys + 1) / solverOp.gas
+                * (uint256(_aData.auctionWins) + 1)
+                / (uint256(_aData.auctionWins + _aData.auctionFails) + solverCount ** 2 + 1) // as solverCount increases,
+                // the dilution of thin auction history increases.
```

Comment:
I can’t tell exactly what this comment means

Reply:
Ok so the more solverOps already queued the harder to score high. Doesn’t this create a latency race because early bidders score higher than late bidders?

Reply:
Sorry not a latency race, but this addition adds the PGA as now a third configuration when bidding. If I want to get a higher score it may be advantageous to dedicate some of my bid amount to the PGA so I can land first.